packages/base/src/3dview/mainview.tsx

```diff
@@ -138,10 +142,24 @@ export class MainView extends React.Component<IProps, IStates> {
         this.lookAtPosition(customEvent.detail.objPosition);
       }
     });
+    this._transformControls.rotationSnap = THREE.MathUtils.degToRad(
+      this.state.rotationSnapValue
+    );
+    this._transformControls.addEventListener('change', () => {
```

Comment:
would this callback be called on every mouse movement?

Reply:
Done!